lib/bunch/enum.ex

```diff
@@ -380,4 +380,28 @@ defmodule Bunch.Enum do
     acc = h |> Tuple.to_list() |> Enum.zip(acc) |> Enum.map(fn {t, r} -> [t | r] end)
     do_unzip(t, acc)
   end
+
+  @spec duplicates(Enum.t(), pos_integer) :: list()
+  @doc """
+  Returns elements that occur at least `min_occurences` times in enumerable.
+
+  Results are ordered descending by the index of `min_occurences`-th duplicate.
```

Comment:
1. This sentence is really hard to understand
2. When you do understand it, you realise that this ordering doesn't make any sense. Please at least reverse it, but i would consider sorting by the number of occurrences